libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -80,7 +111,7 @@ class EmbeddingStore(Base):
             custom_id = Column(VARCHAR, nullable=True)  # column for user defined ids.
             query_metadata = Column(JSON, nullable=True)
             query = Column(NVARCHAR, nullable=False)  # defaults to NVARCHAR(MAX)
-            embeddings = Column(VARBINARY, nullable=False)
+            embeddings = Column(VARBINARY(self._embedding_length), nullable=False)
```

Comment:
Even though currently different length of embeddings is ok because of the usage of VARBINARY type, in future since we know, it will error out if vector lengths dont match, We should start returning error right now. This will avoid a regression - something thats working now - will not be working in future by design.